llvm/lib/DebugInfo/LogicalView/Readers/LVELFReader.cpp

```diff
@@ -415,6 +415,8 @@ void LVELFReader::processOneAttribute(const DWARFDie &Die, LVOffset *OffsetPtr,
       if (FoundLowPC) {
         if (CurrentLowPC == MaxAddress)
           CurrentElement->setIsDiscarded();
+        // Consider the case of WebAssembly.
```

Comment:
Should the name of this file/class be `LVDWARFReader` instead of ELF?

Reply:
That is a very good point.
I think `LVDWARFReader` would match the another reader `LVCodeViewReader` as they will reflect the type of debug information format that they are parsing.

@dwblaikie, @adrian-prantl, @pogo59: any thoughts on this proposal?

Reply:
I agree. Although the renaming should be done as its own separate NFC patch.

Reply:
I will prepare a separate NFC patch with the following changes:
Filename change: `LVElfReader.cpp[h]`  ->  `LVDWARFReader.cpp[h]`
Class name change: `LVELFReader`  ->  `LVDWARFReader`

Reply:
Does this mean this whole file will be renamed, or only the wasm-related parts will be spun off as a new file? The comment looks like the whole file/class will be renamed.. Then does this class not support ELF anymore? The file heading says it supports ELF/MachO. (Should we update that too?)